src/Microsoft.Health.Fhir.Core/Features/Conformance/IConformanceProvider.cs

```diff
@@ -6,13 +6,13 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using Hl7.Fhir.Model;
+using Hl7.Fhir.ElementModel;
 
 namespace Microsoft.Health.Fhir.Core.Features.Conformance
 {
     public interface IConformanceProvider
     {
-        Task<CapabilityStatement> GetCapabilityStatementAsync(CancellationToken cancellationToken = default(CancellationToken));
+        Task<ITypedElement> GetCapabilityStatementAsync(CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
I'm thinking we stick with `ResourceElement` as the abstraction for Resources

Reply:
Yes!  Updated.